### DISCUSSION TITLE
Welcome to HDMIA Discussions Forum!

### DISCUSSION THREAD

**Discussion by magedhelmy1** [magedhelmy1/CapillaryDetection#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 21317389** [comment]
Hi, I'm honored to use this code.
But when I install docker and run it, I don't know how to run GUI.
If possible,maybe you can share the detail step for me.
Thank you!!
